Marlin/Marlin_main.cpp

```diff
@@ -789,8 +789,48 @@ extern "C" {
   extern void digipot_i2c_init();
 #endif
 
+inline void echo_command(char * const cmd) {
```

Comment:
Is it not echoing correctly? The intention is that the `cmd` pointer not be altered in the function. (and `char * const` is a _constant pointer_ to a `char`).